src/ng/directive/select.js

```diff
@@ -76,9 +76,11 @@ var SelectController =
     }
   };
 
+  var scopeDestroyed = false;
```

Comment:
If `$$destroyed` would work then I would prefer that. It is one less thing to store in memory and keep track of in maintenance of the code.